qiita_ware/test/test_commands.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
```

Comment:
Can you remove this line? Since our files can't be executable (thanks :nose:tests), the shebang has no use.